metadata-editor/conf/routes

```diff
@@ -20,6 +20,12 @@ GET     /metadata/:id/usage-rights                      controllers.EditsControl
 PUT     /metadata/:id/usage-rights                      controllers.EditsController.setUsageRights(id: String)
 DELETE  /metadata/:id/usage-rights                      controllers.EditsController.deleteUsageRights(id: String)
 
+# Collections
+GET     /collections                                    controllers.CollectionsController.getCollections
+POST    /collections                                    controllers.CollectionsController.addCollectionFromJson
+PUT     /collections/*collection                        controllers.CollectionsController.addCollectionFromString(collection: String)
```

Comment:
This shouldn't really be a PUT as it doesn't adhere to the HTTP spec of replacing the resource at the given URI with the provided entity.